Lib/test/test_sys.py

```diff
@@ -693,7 +693,7 @@ def test_43581(self):
 
     def test_intern(self):
         self.assertRaises(TypeError, sys.intern)
-        s = "never interned before" + str(random.random())
+        s = "never interned before" + str(random.int(10**8, 10**9))
```

Comment:
random.int doesn't exist. What's the point of putting a minimum at 10**8? Why not 0?

Reply:
There is a more recent commit in the PR fixed the bug. The minimal start is for ensure the random parts have 9 digits, I think it's more consistent.